src/Symfony/Component/Notifier/Transport/NullTransport.php

```diff
@@ -34,6 +34,10 @@ public function __construct(EventDispatcherInterface $dispatcher = null)
 
     public function send(MessageInterface $message): SentMessage
     {
+        if (null === $message->getTransport()) {
+            $message->transport((string) $this);
```

Comment:
I needed to make sure that every transport has a name to show this name in the profiler panel.